pom.xml

```diff
@@ -77,21 +74,14 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-weekly</artifactId>
-                <version>3387.v0f2773fa_3200</version>
+                <artifactId>bom-2.462.x</artifactId>
```

Comment:
https://github.com/jenkinsci/bom/releases/tag/3482.vc10d4f6da_28a_ can now use the 2.479.x BOM.

Reply:
Done in https://github.com/jenkinsci/workflow-job-plugin/pull/470/commits/1232095a5da8aa49b00ed6902dac7dbafb20c508.